stdlib/public/runtime/Exclusivity.cpp

```diff
@@ -38,14 +38,22 @@
 #elif _MSC_VER
 #include <intrin.h>
 #define get_return_address() _ReturnAddress()
+#elif defined(__wasm__)
+#define get_return_address() ((void*) 0)
 #else
 #error missing implementation for get_return_address
 #define get_return_address() ((void*) 0)
 #endif
 
 using namespace swift;
 
+#if defined(__wasm__)
```

Comment:
Should this not be `__wasi__`, because this is not related to the object file format?